src/taipy/core/config/data_node_config.py

```diff
@@ -294,14 +335,16 @@ def _configure_default(storage_type: str, scope: Scope = _DEFAULT_SCOPE, cacheab
         Returns:
             `DataNodeConfig^`: The default data node configuration.
         """
-        section = DataNodeConfig(_Config.DEFAULT_KEY, storage_type, scope, cacheable, **properties)
+        section = DataNodeConfig(
+            _Config.DEFAULT_KEY, storage_type, scope, cacheable, **properties
+        )
         Config._register(section)
         return Config.sections[DataNodeConfig.name][_Config.DEFAULT_KEY]
 
     @staticmethod
     def _configure(
         id: str,
-        storage_type: str = _DEFAULT_STORAGE_TYPE,
+        storage_type: Optional[str] = None,
```

Comment:
Just out of curiosity, but why is it None for storage type here? Since we're only providing limited options for the dn, would it be better to have a default value like pickle instead?

Reply:
> 

The issue here is that if you make the default `"pickle"` in the `_configure()` method, then it ignores the default set in `configure_default_data_node()`.   Using `None` makes it look at whatever was set in `configure_default_data_node()`

I updated the tests to test both the default without calling `configure_default_data_node()` and with calling it.